### ISSUE TITLE
[question]: Create Player example returns a 400

### ISSUE THREAD

**Issue by Kieran-McC** [OneSignal/onesignal-go-api#12]
### How can we help?

**PROBLEM**
When I try to follow the Create Player example I am getting a 400 error.  See [here](https://github.com/OneSignal/onesignal-go-api/blob/main/docs/DefaultApi.md#createplayer)

**CODE**
Below is the code sample that I am following. 
I have a one signal app that is hooked up to a firebase application using the FCM creds. 

```
package main

import (
    "context"
    "fmt"
    "os"

    "github.com/OneSignal/onesignal-go-api"
)

func main() {
    appId := "appId_example" // string | 
    notificationId := "notificationId_example" // string | 

    configuration := onesignal.NewConfiguration()
    apiClient := onesignal.NewAPIClient(configuration)

    appAuth := context.WithValue(context.Background(), onesignal.AppAuth, "APP_KEY_STRING")

    resp, r, err := apiClient.DefaultApi.CancelNotification(appAuth, notificationId).AppId(appId).Execute()

    if err != nil {
        fmt.Fprintf(os.Stderr, "Error when calling `DefaultApi.CancelNotification``: %v\n", err)
        fmt.Fprintf(os.Stderr, "Full HTTP response: %v\n", r)
    }
    // response from `CancelNotification`: InlineResponse2001
    fmt.Fprintf(os.Stdout, "Response from `DefaultApi.CancelNotification`: %v\n", resp)
}
```

In place of `APP_KEY_STRING` I am using the OneSignal App ID from the Keys and IDs section of my app. 
Is that what I should be using here? I tried it with the REST API Key but with no luck either. 

**RESPONSE**
The response I get is:
`Error when calling `DefaultApi.CreatePlayer``: 400 Bad Request`

The full repsonse is:
```
Full HTTP response: &{400 Bad Request 400 HTTP/2.0 2 0 map[Access-Control-Allow-Headers:[SDK-Version] Access-Control-Allow-Origin:[*] Alt-Svc:[h3=":443"; ma=86400, h3-29=":443"; ma=86400] Cache-Control:[no-cache] Cf-Cache-Status:[DYNAMIC] Cf-Ray:[71fec6fbc9b6a1de-YYZ] Content-Type:[application/json; charset=utf-8] Date:[Thu, 23 Jun 2022 16:59:58 GMT] Expect-Ct:[max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"] Referrer-Policy:[strict-origin-when-cross-origin] Server:[cloudflare] Status:[400 Bad Request] Vary:[Accept-Encoding] Via:[1.1 google] X-Content-Type-Options:[nosniff] X-Download-Options:[noopen] X-Envoy-Upstream-Service-Time:[6] X-Frame-Options:[SAMEORIGIN] X-Permitted-Cross-Domain-Policies:[none] X-Request-Id:[ec840f00-d31d-4826-8dd7-a461c7444ac8] X-Runtime:[0.003259] X-Xss-Protection:[1; mode=block]] {{"errors":["app_id not found. You may be missing a Content-Type: application/json header."]}} -1 [] false true map[] 0xc0003e6300 0xc0004ca0b0}
Response from `DefaultApi.CreatePlayer`: <nil>
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by jfishman1** [comment]
Hi @Kieran-McC for 400 errors, you are likely missing some required properties or malformed variable values. We would like to see all the properties and variable values used. Can you reach out to support@onesignal.com with this data so we can better understand and reproduce?

**Comment by rgomezp** [comment]
Howdy Kieran,
The issue regarding the `name` param should now be resolved in v 1.0.4. Thanks for your patience.
